openmc/statepoint.py

```diff
@@ -13,17 +13,28 @@
 
 
 class SourceSite(object):
+    """A single source site, usually resulting from fission.
```

Comment:
(n,xn) reactions actually do not create source sites that would end up in a statepoint. Right now, e.g., (n,2n) reactions simply just double the weight of the incoming particle. I'll change this to remove the 'usually' which really means 'always'.